scripts/release/mule/common/ostype.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Huh...I'm not sure this is necessarily better than the other ostype.sh script at this point. 🤔